pkg/vere/main.c

```diff
@@ -2213,6 +2245,7 @@ _cw_play(c3_i argc, c3_c* argv[])
 
   //  XX handle SIGTSTP so that the lockfile is not orphaned?
   //
+  u3_Host.eve_d = u3m_boot(u3_Host.dir_c, (size_t)1 << u3_Host.ops_u.lom_y);
```

Comment:
This won't restore the right snapshot on a full replay, or on one where `chk` is empty or missing.